JitsiConnection.ts

```diff
@@ -62,7 +69,7 @@ export default class JitsiConnection {
         // Initialize the feature flags so that they are advertised through the disco-info.
         FeatureFlags.init(options.flags || {});
 
-        this.xmpp = new XMPP(options, token);
+        this._xmpp = new XMPP(options, token);
```

Comment:
You don't need to access it like this, leave it as `this.xmpp` since it's a getter.